### PR TITLE
Enable HTTPS Everywhere on Vox Media properties

### PR DESCRIPTION
**Pull request by iamcarrico**
Howdy! 

I am not entirely sure if all of these are 100% correct— but the following sites all have HTTPS enabled on them, and we should probably have them all enabled by default now. Let me know if there is any other information I can help provide.

Sites enabled:

* vox.com
* theverge.com
* eater.com
* racked.com
* curbed.com
* polygon.com
* recode.net

I followed the PR pattern from #9326, but am happy to spend some time making improvements as requested. 

I have been waiting and wanting to do a PR like this for some time... I hope we can make good things happen

### CODE REVIEW

File: src/chrome/content/rules/Eater.xml [hunk 1]
```diff
@@ -24,15 +24,13 @@
...
 				-->
 	<target host="www.eater.com" />
-
-	<!--	Complications:
-				-->
 	<target host="eater.com" />
+	<target host="*.eater.com" />
 
 
 	<rule from="^http://eater\.com/"
```
Comment (J0WI) [inline src/chrome/content/rules/Eater.xml]: This redirect is not necessary.

File: src/chrome/content/rules/Eater.xml [hunk 2]
```diff
@@ -24,15 +24,13 @@
 		- Fonts on www from fonts.voxmedia.com
 
 -->
-<ruleset name="Eater.com (partial)">
+<ruleset name="Eater.com">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="www.eater.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Eater.xml]: the wildcard target makes this obsolete.

File: src/chrome/content/rules/Eater.xml [hunk 3]
```diff
@@ -24,15 +24,13 @@
...
-
-	<!--	Complications:
-				-->
 	<target host="eater.com" />
+	<target host="*.eater.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Eater.xml]: We require at least 3 test urls for wildcards. Please add as many different hosts as you can.

Comment (J0WI) [inline src/chrome/content/rules/Eater.xml]: Please have a look at the comments on the top. You may update/remove them if all hosts are working now.

File: src/chrome/content/rules/Curbed.com.xml [hunk 4]
```diff
@@ -50,52 +50,23 @@
...
 -->
-<ruleset name="Curbed.com (partial)">
+<ruleset name="Curbed.com">
 
 	<!--	Direct rewrites:
 				-->
-	<target host="hamptons.curbed.com" />
-	<target host="ny.curbed.com" />
-	<target host="www.ny.curbed.com" />
+	<target host="www.curbed.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Curbed.com.xml]: The wildcard target makes this obsolete, but you should add `curbed.com`.

File: src/chrome/content/rules/Curbed.com.xml [hunk 5]
```diff
@@ -50,52 +50,23 @@
...
+	<target host="*.curbed.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Curbed.com.xml]: Please also update the comment section on the top.

File: src/chrome/content/rules/Polygon.com.xml [hunk 6]
```diff
@@ -16,57 +16,12 @@
...
-		<test url="http://www.polygon.com/pages/about" />
-
-	<!-- Fix #4007 -->
-	<exclusion pattern="http://www\.polygon\.com/chorus_page/optimally_sized_images" />
-	
-	<test url="http://www.polygon.com/chorus_page/optimally_sized_images" />
-
 	<!--	Possible trackers (untested):
 						-->
 	<securecookie host="^\.polygon\.com$" name="^__[qu].*" />
```
Comment (J0WI) [inline src/chrome/content/rules/Polygon.com.xml]: The ^ to www redirect is not necessary.

File: src/chrome/content/rules/Recode.net.xml [hunk 7]
```diff
@@ -9,140 +9,11 @@
 	ᶜ See https://owasp.org/index.php/SecureFlag
 
 -->
-<ruleset name="Recode.net (partial)">
+<ruleset name="Recode.net">
 
 	<target host="recode.net" />
 	<target host="www.recode.net" />
```
Comment (J0WI) [inline src/chrome/content/rules/Recode.net.xml]: Please also add `events.recode.net`

File: src/chrome/content/rules/Racked.com.xml [hunk 8]
```diff
@@ -7,7 +7,7 @@
 
 	* CORS issues, https://github.com/EFForg/https-everywhere/issues/7381
 -->
-<ruleset name="Racked.com" default_off="CORS issues">
```
Comment (J0WI) [inline src/chrome/content/rules/Racked.com.xml]: Please also update the comments.

File: src/chrome/content/rules/Racked.com.xml [hunk 9]
```diff
@@ -7,7 +7,7 @@
...
+<ruleset name="Racked.com">
 	<target host="racked.com" />
 	<target host="www.racked.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Racked.com.xml]: There are some more subdomains. Could you add them too?

File: src/chrome/content/rules/Vox.com.xml [hunk 10]
```diff
@@ -23,30 +23,21 @@
...
-<ruleset name="Vox.com" default_off="redirects to http">
+<ruleset name="Vox.com">
 
 	<!--	Direct rewrites:
 				-->
 	<target host="www.vox.com" />
-
-	<!--	Complications:
-				-->
 	<target host="vox.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Vox.com.xml]: See also:
`conversations.vox.com`
`votecompass.vox.com`

File: src/chrome/content/rules/Vox.com.xml [hunk 11]
```diff
@@ -23,30 +23,21 @@
...
 	<!--	Possible trackers (untested):
 						-->
 	<securecookie host="^\." name="^__[qu].*" />
 
 
+	<test url="http://www.vox.com/2016/1/7/10726296/wheres-rey-star-wars-monopoly" />
+	<test url="http://vox.com/2016/1/7/10726296/wheres-rey-star-wars-monopoly" />
+
 	<rule from="^http://vox\.com/"
 		to="https://www.vox.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Vox.com.xml]: The ^ to www redirect is not necessary.

File: src/chrome/content/rules/Vox_Media.com.xml [hunk 12]
```diff
@@ -51,7 +51,7 @@
 	ᶜ See https://owasp.org/index.php/SecureFlag
 
 -->
-<ruleset name="Vox Media.com (partial)">
+<ruleset name="Vox Media.com=">
```
Comment (J0WI) [inline src/chrome/content/rules/Vox_Media.com.xml]: I think that's a typo.

Comment (J0WI) [inline src/chrome/content/rules/Vox_Media.com.xml]: Please also update the comment section.

**Comment by J0WI** [comment]
There are a lot of changes to review, I would prefer to have one PR for each file.

**Comment by vuiiiiuc** [comment]
@iamcarrico Did you look into J0WI's review of your PR?

**Comment by iamcarrico** [comment]
Yeah, sorry— just behind here. I will get them all in there. I just got access to all of our DNS information to ensure that I can add in the appropriate tests.

**Comment by J0WI** [comment]
@iamcarrico can you share any news about the progress here?

**Comment by iamcarrico** [comment]
Slow. Just not able to find a lot of spare time right now. I just pushed the work from the past couple weeks— I am also working to ensure the `securecookie` changes can be pushed into our production site as soon as possible.

**Comment by jeremyn** [comment]
@iamcarrico If you don't plan on working on this in the foreseeable future, please close the PR. Also, in any case it's best if PRs focus on just one file each if possible, so if you do want to update these rulesets in the future, you should make one PR for `curbed.com`, another for `eater.com` and so on.

**Comment by jeremyn** [comment]
@iamcarrico Also, as a general clean-up action, we'll close this PR soon if we don't hear from you.

**Comment by jeremyn** [comment]
I'm closing this pull request due to lack of response from @iamcarrico .

**Pull request closed without merging**